components/eventing-controller/pkg/handlers/utils.go

```diff
@@ -80,7 +80,7 @@ func getQos(qosStr string) (types.Qos, error) {
 func getInternalView4Ev2(subscription *eventingv1alpha1.Subscription, apiRule *apigatewayv1alpha1.APIRule, defaultWebhookAuth *types.WebhookAuth, defaultProtocolSettings *eventingv1alpha1.ProtocolSettings, defaultNamespace string) (*types.Subscription, error) {
 	emsSubscription, err := getDefaultSubscription(defaultProtocolSettings)
 	if err != nil {
-		return nil, errors.Wrap(err, "failed to apply default protocol settings")
+		return nil, errors.Wrap(err, "apply default protocol settings failed")
 	}
```

Comment:
Just a last comment on these kind of message changes. I think the previous messages are grammatically correct. Why have you decided to make these changes? Is it some sort of logging convention?

Reply:
As discussed, we'll leave it like that to be consistent with other log messages.